_includes/subscribe.html

```diff
@@ -49,12 +50,16 @@
       id="mc-embedded-subscribe-form"
       name="mc-embedded-subscribe-form"
       class="validate"
-      onsubmit="document.getElementById(mc-embedded-subscribe-form).submit();"
+      onsubmit="return validateForm(event);"
     >
       <div id="mc_embed_signup_scroll">
         <div class="subscribe-inputbox">
-          <input type="text" value="" placeholder="First Name" name="FNAME" class="NAME" id="mce-FNAME" required />
-          <input type="text" value="" placeholder="Last Name" name="LNAME" class="NAME" id="mce-LNAME" required />
+          <input type="text" value="" placeholder="First Name" name="FNAME" class="NAME" id="mce-FNAME" aria-label="first name input" required/>
+          <input type="text" value="" placeholder="Last Name" name="LNAME" class="NAME" id="mce-LNAME" aria-label="last name input" required />
+        </div>
+        <div class="subscribe-container">
+          <span id="mce-FNAME-error" class="subscribe-container-error"></span>
+          <span id="mce-LNAME-error" class="subscribe-container-error"></span>
```

Comment:
Using the aria-describedby attribute on the input elements to associate them with their respective error messages would help screen readers read out the error message when the input elements are in focus.

Reply:
Something like this:
<input type="text" value="" placeholder="First Name" name="FNAME" class="NAME" id="mce-FNAME" aria-label="first name input" required aria-describedby="mce-FNAME-error"/>
```suggestion
          <input type="text" value="" placeholder="First Name" name="FNAME" class="NAME" id="mce-FNAME" aria-label="first name input" required aria-describedby="mce-FNAME-error"/>
          <input type="text" value="" placeholder="Last Name" name="LNAME" class="NAME" id="mce-LNAME" aria-label="last name input" required aria-describedby="mce-LNAME-error" />
        </div>
        <div class="subscribe-container">
          <span id="mce-FNAME-error" class="subscribe-container-error"></span>
          <span id="mce-LNAME-error" class="subscribe-container-error"></span>
```